### PR TITLE
[Remove unused reference] LPS-165884

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to remove unused fields in `document-library-service`. Thanks!

### CODE REVIEW

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/LiferayVersioningCapability.java [hunk 1]
```diff
@@ -223,9 +222,6 @@ private FileEntry _purgeVersions(
 		return fileEntry;
 	}
 
-	@Reference
-	private DLConfiguration _dlConfiguration;
```
Comment (dantewang) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/LiferayVersioningCapability.java:227]: @JanisZhang This commit did remove the usage to the field, but added `@Reference` annotation to it. This is strange as we usually don't declare dependency on a configuration this way. Instead, we use `configurationPid` property in the `@Component` annotation.

I think this field can be removed, as it has a 1..1 reference to `VersionPurger` which depends on the config. But please modify your commit message to be accurate.

**Review by dantewang — CHANGES_REQUESTED** [review]
@JanisZhang Please update commit message as commented, thanks!

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/LiferayVersioningCapability.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/repository/capabilities/LiferayVersioningCapability.java:227]: @dantewang Got it. THX

**Comment by JanisZhang** [comment]
@dantewang @zhangyan0701  Force-pushed. I only changed the commit message, it shouldn't affect the test results.

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by dantewang** [comment]
ci:test:relevant

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
Approved. Please also see https://github.com/liferay-core-infra/liferay-portal/pull/2167#issuecomment-1290027735 for extra context.

**Comment by zhangyan0701** [comment]
No regressions found.

**Comment by dantewang** [comment]
PR sent to https://github.com/brianchandotcom/liferay-portal/pull/125250

cc @shuyangzhou @tinatian

**Pull request closed without merging**